sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -26,7 +26,6 @@
   },
   "files": [
     "LICENSE.txt",
-    "ThirdPartyNotices.txt",
```

Comment:
There is no browser entry to remove as certificates was missing it in the first place :)